projects/core/src/routing/store/reducers/router.reducer.spec.ts

```diff
@@ -86,6 +86,7 @@ describe('Router Reducer', () => {
           params: {},
           context: { id: 'homepage' },
           cmsRequired: true,
+          routeName: '',
```

Comment:
let's remove this one, it's a left over from me. Sorry about that.

Reply:
done